iceoryx_posh/source/roudi/roudi_process.cpp

```diff
@@ -550,32 +550,30 @@ void ProcessManager::addApplicationForProcess(const ProcessName_t& name) noexcep
     }
 }
 
-void ProcessManager::addRunnableForProcess(const ProcessName_t& processName,
-                                           const RunnableName_t& runnableName) noexcept
+void ProcessManager::addNodeForProcess(const ProcessName_t& processName, const NodeName_t& nodeName) noexcept
 {
     std::lock_guard<std::mutex> g(m_mutex);
 
     RouDiProcess* process = getProcessFromList(processName);
     if (nullptr != process)
     {
-        runtime::RunnableData* runnable =
-            m_portManager.acquireRunnableData(ProcessName_t(cxx::TruncateToCapacity, processName),
-                                              RunnableName_t(cxx::TruncateToCapacity, runnableName));
+        runtime::NodeData* node = m_portManager.acquireNodeData(ProcessName_t(cxx::TruncateToCapacity, processName),
+                                                                NodeName_t(cxx::TruncateToCapacity, nodeName));
 
-        auto offset = RelativePointer::getOffset(m_mgmtSegmentId, runnable);
+        auto offset = RelativePointer::getOffset(m_mgmtSegmentId, node);
 
         runtime::MqMessage sendBuffer;
-        sendBuffer << runtime::mqMessageTypeToString(runtime::MqMessageType::CREATE_RUNNABLE_ACK)
-                   << std::to_string(offset) << std::to_string(m_mgmtSegmentId);
+        sendBuffer << runtime::mqMessageTypeToString(runtime::MqMessageType::CREATE_NODE_ACK) << std::to_string(offset)
+                   << std::to_string(m_mgmtSegmentId);
 
         process->sendToMQ(sendBuffer);
-        m_processIntrospection->addRunnable(ProcessName_t(cxx::TruncateToCapacity, processName.c_str()),
-                                            RunnableName_t(cxx::TruncateToCapacity, runnableName.c_str()));
-        LogDebug() << "Created new runnable " << runnableName << " for application " << processName;
+        m_processIntrospection->addNode(ProcessName_t(cxx::TruncateToCapacity, processName.c_str()),
+                                        NodeName_t(cxx::TruncateToCapacity, nodeName.c_str()));
+        LogDebug() << "Created new node " << nodeName << " for application " << processName;
```

Comment:
Maybe we can rename it to "unknown process" to make the message consistent to the code.